### DISCUSSION TITLE
Looking for 64 bit dlls of CefSharp for our 64 bit application

### DISCUSSION THREAD

**Discussion by susmitakanchinadham** [cefsharp/CefSharp#4100] [Q&A]
I'm looking for 64 bit dlls of the below CefSharp libraries for our 64 bit application.
CefSharp
CefSharp.Core
CefSharp.WinForms
CefSharp.Wpf
Could you please point me to them.
Thanks!

**Comment by amaitland** [comment]
Those dlls are all compiled as AnyCPU(MSIL), you don't need a x64 version.

**Reply by susmitakanchinadham** [reply]
But, I'm facing error like "CS8012	Referenced assembly 'CefSharp, Version=84.4.10.0, Culture=neutral' targets a different processor." while integrating with my 64 bit application.
Do you think it is due to old version of CEF dlls, where they were compiled in x86 platform.

**Reply by amaitland** [reply]
The Nuget packages were restructure in version 87 see  #3319

**Reply by susmitakanchinadham** [reply]
I updated the versions.Thanks a lot for your inputs!
